view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -190,13 +190,7 @@ define(
                                     self.installments(0);
                                 }
                             }
-
-                            // for BCMC as this is not a core payment method inside magento use maestro as brand detection
-                            if (creditCardType == "BCMC") {
-                                self.creditCardType("MI");
-                            } else {
-                                self.creditCardType(creditCardType);
-                            }
+                            self.creditCardType(creditCardType);
```

Comment:
looks good, maybe a follow up ticket would be nice to update it the card type based on the payments or the payments/details response (IF available) :)